addons/crm/static/src/js/tours/crm.js

```diff
@@ -8,93 +8,117 @@ var _t = core._t;
+    content: _t('Look for a Contact or create a new one.<br><i>Tip : Did you know you can search by VAT Number as well?</i>'),
+    position: "right",
+    run: function (actions) {
+        actions.text("Brandon Freeman", this.$anchor.find("input"));
+    },
+}, {
+    trigger: ".o_kanban_quick_create input[name='name']",
     content: _t("<b>Choose a name</b> for your opportunity, example: <i>'Need a new website'</i>"),
     position: "right",
+}, {
+    trigger: '.o_kanban_quick_create .o_field_monetary[name="planned_revenue"]',
+    content: _t("Define here the Expected Revenue of this Opportunity."),
+    position: 'right',
 }, {
     trigger: ".o_kanban_quick_create .o_kanban_add",
-    content: _t("Click here to <b>add your opportunity</b>."),
+    content: _t("Now, <b>add your Opportunity</b> to your Pipeline."),
     position: "bottom",
 }, {
     trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child",
     content: _t("<b>Drag &amp; drop opportunities</b> between columns as you progress in your sales cycle."),
-    position: "right",
+    position: "bottom",
     run: "drag_and_drop .o_opportunity_kanban .o_kanban_group:eq(2) ",
 }, {
     trigger: ".o_kanban_record:not(.o_updating) .o_activity_color_default",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("This opportunity has <b>no activity planned</b>."),
-    position: "bottom"
+    content: _t("Looks like nothing is planned. :(<br><i>Tip : Schedule activities to keep track of everything you have to do!</i>"),
+    position: "bottom",
 }, {
     trigger: ".o_schedule_activity",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("Let's schedule an activity."),
+    content: _t("Let's <b>schedule an activity.</b>"),
     position: "bottom"
 }, {
-    trigger: '.modal-body .o_field_many2one',
+    trigger: '.modal-body .o_field_char[name="summary"]',
+    extra_trigger: ".o_opportunity_kanban",
+    content: _t("Let’s do a follow-up next week."),
+    position: "bottom",
+}, {
+    trigger: '.modal-body .o_field_date',
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("Choose an activity type.<br/>You can customize them in the general settings."),
+    content: _t("Choose a deadline."),
+    position: "right",
+}, {
+    trigger: '.modal-footer button[name="action_close_dialog"]',
+    content: _t("All set. Let’s <b>Schedule</b> it."),
+    position: "bottom",
+    run: function (actions) {
+        actions.auto('.modal-footer button[special=cancel]');
+    },
+}, {
+    trigger: ".o_opportunity_kanban .o_kanban_record:last-child",
+    content: _t("Drag your opportunity to <b>Won</b> when you get the deal. Congrats !"),
+    position: "bottom",
+    run: "drag_and_drop .o_opportunity_kanban .o_kanban_group:eq(3) ",
+}, {
+    trigger: ".o_button_generate_leads",
+    content: _t("Looking for more opportunities ? Try the <b>Lead Generation</b> tool."),
+    position: "bottom",
+    run: function (actions) {
```

Comment:
Hello,

If you look at bit at module organization, lead generation is an optional module. If you install only crm, you will ran into troubles. You will have to make a base tour in crm, and add steps in crm_iap_lead .